app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -542,19 +542,36 @@
   </div>
   <br>
   <div class="{{unless device.isMobile 'right floated large' 'mini three'}} ui fields buttons">
-    <button class="ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'moveForward'}}>
-      {{t 'Forward'}}
-      <i class="right chevron icon"></i>
-    </button>
-    <button class="blue ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'saveDraft'}}>
-      {{t 'Save draft'}}
-      <i class="save icon"></i>
-    </button>
-    {{#if data.event.locationName}}
-      <button class="green ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'publish'}}>
-        {{t 'Publish'}}
-        <i class="check icon"></i>
+    {{#if (eq data.verifiedStatus true)}}
```

Comment:
you don't need to compare anything with true. if (true == true) is a redundant comparison.

Reply:
Oh, so `{{#if data.verifiedStatus }}` will be suffice

Reply:
yes, @uds5501

Reply:
Aslo change the name from `verifiedStatus` to `isUserVerified` @uds5501

Reply:
understood!

Reply:
@CosmicCoder96 Did the changes